index.html

```diff
@@ -0,0 +1,139 @@
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box7">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box8">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box9" style="min-height: 16px;">
+          <img class="globalsearch" src="./img/globalsearch_field.png" alt="">
+      </div>
+    </div>
+    <header class="main-header">
+      <section class "main-text">
+        <img id="display-text1" src="./img/your-verse-small.png" alt="what is your verse">
+      </section>
+      <section class="sub-header">
+        <img src="./img/everyone-share.png" alt="Everyone Share">
+      </section>
+      <div class="watch-button">
+        <a class="watch-anchor" href="#">
+          <img class="watch-img" src="./img/watch-film-btn.png" alt="Watch the film">
+        </a>
+      </div>
+    </header>
+    <section class="promos">
+      <div class="promo promo1">
+        <img src="./img/promo1.jpg" alt="">
+      </div>
+      <div class="promo promo2">
+        <img src="./img/promo2.jpg" alt="">
+      </div>
+      <div class="promo promo3">
+        <img src="./img/happy_birthday.jpg" alt="">
+      </div>
+      <div class="promo promo4">
+        <img src="./img/supplier_resp.jpg" alt="">
+      </div>
+    </section>
+  </main>
+  <footer class="mainfooter">
+    <div class="menu-footer_left">
+      <p>Shop the <a href="#">Apple Retail Store</a>,
+        or find a <a href="#">reseller</a>.</p>
+      <p class="copyright">Copyright © 2014 Apple Inc. All rights reserved.
+        <ul class="menu-left">
+          <li>
+            <a href="#">Terms of Use </a>
+          </li>
+          <li>
+            <a href="#">Privacy Policy </a>
+          </li>
+        </ul>
+      </p>
+    </div>
+    <div class="menu-footer-right">
+      <ul class="menu-right">
+        <li>
+          <a class="menu-right-opt" href="#">Site Map</a></li>
+        <li>
+          <a class="menu-right-opt" href="#">Hot News</a>
+        </li>
+        <li>
+          <a class="menu-right-opt" href="#">RSS Feeds</a>
+        </li>
+        <li>
+          <a class=="menu-right-opt" href="#">Media Info</a>
+        </li>
+        <li>
+          <a class="menu-right-opt" href="#">Environment</a>
+        </li>
+        <li>
+          <a class="menu-right-opt" href="#">Job Opportunities</a>
+        </li>
+        <li>
+          <a class="menu-right-opt" href="#">Contact Us</a>
+        </li>
+        <li class="worldwide">
+          <a href="#"><img src="./img/flag-btn.png" alt="Choose your country or region" height="16" width="16"></a>
```

Comment:
:memo: please put **heigh="16"** and **width="16"** in your CSSfile

Reply:
done